docs/source/ui/pages/run_management.md

```diff
@@ -189,3 +199,155 @@ Use the `Settings` tab to delete or archive the single run
 
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Archive card" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/archive_card.png">
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Unarchive card" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/unarchive_card.png">
+
+## Experiment page
+
+The experiment page provides overall information about a single experiment and helps to explore the runs attached to the experiment. 
+Here are the tabs available on the experiment page:
+
+- [Overview](#id21)
+- [Runs](#id24)
+- [Notes](#id25)
+- [Settings](#id26)
+
+### Overview
+
+The Overview tab provides an overview of the runs attached to the experiment. 
+It displays general information such as their status, progress, and other relevant metadata.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Experiment page overview tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/experiment-page/experiment-page-overview-tab.png">
+
+#### Statistics
+
+On this page, you can view the total number of runs, as well as the number of runs that are `archived`, `active`, and `finished`.
+You can click on the corresponding cards to view more information about these runs, and you can also use these cards to navigate to the [Runs explorer](../pages/run_management.html#runs-explorer) page with a specific query.
+The statistics bar, below the `status` cards, displays the distribution of runs by status.
+
+#### Contributions calendar
+
+The contributions calendar represents the intensity of executed runs during the previous year. More on this read [here](../pages/home_page.html#id8).
```

Comment:
maybe worth adding just one more sentence to explain what intensity stands for and then only refer to more details?